sandy/decay.py

```diff
@@ -182,55 +183,132 @@ def get_br(self, with_uncertainty=True):
 
         Returns
         -------
-        `pandas.DataFrame` or `pandas.Series`
-            dataframe with branching ratio and associated uncertainty
-            series with branching ratio if with_uncertainty = False
+        `pandas.DataFrame`
+            dataframe with decay mode, branching ratio and associated uncertainty or
+            dataframe with decay mode and branching ratio if with_uncertainty = False
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> rdd.get_branching_ratio()
+               decay mode          BR         DBR
+        ZAM                                      
+        922350      alpha 1.00000e+00 1.00000e-04
+        922350         SF 7.20000e-11 2.10000e-11
+        942410      alpha 2.44000e-05 0.00000e+00
+        942410       beta 9.99976e-01 0.00000e+00
+
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> rdd.get_branching_ratio(False)
+               decay mode          BR
+        ZAM                          
+        922350      alpha 1.00000e+00
+        922350         SF 7.20000e-11
+        942410      alpha 2.44000e-05
+        942410       beta 9.99976e-01
+        
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942410, 10010, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> rdd.get_branching_ratio(False)
+               decay mode          BR
+        ZAM                          
+        922350      alpha 1.00000e+00
+        922350         SF 7.20000e-11
+        942410      alpha 2.44000e-05
+        942410       beta 9.99976e-01
+        """
+        br = []
+        zam = []
+        for z, dic in self.data.items():
+            try:
+                for rtyp, dk in dic['decay_modes']:
+                    br.append([
+                        rtyp2decaymode(rtyp),
```

Comment:
I would keep te RTYP like before. What if a nuclide decays beta and emits also 2 neutrons?

Reply:
It works also for multiple particle decay.
In the case that you mentioned the function will return `'beta, n emission, n emission'`.  In this way you avoid having zero that has no meaning in rtyp. Do you mean it's too long?

Reply:
For the moment we don't need this. We can keep the RTYP